labs-ios-starter/Application/View/UI/Custom Objects/StandardButton.swift

```diff
@@ -0,0 +1,158 @@
+// Copyright © 2020 Shawn James. All rights reserved.
+// StandardButton.swift
+
+import UIKit
+
+/// The button styling used for most buttons
+class StandardButton: UIButton {
+
+    // MARK: - Properties
+        
+    /// used to show loading animation
+    lazy var activityIndicator: UIActivityIndicatorView = {
+        let indicator = UIActivityIndicatorView()
+        indicator.hidesWhenStopped = true
+        indicator.color = .white
```

Comment:
do we want to use the system background color instead so it's white normally and reversed in dark mode?